config/kustomization.yaml

```diff
@@ -18,22 +18,22 @@ namePrefix: cluster-api-provider-baremetal-
 # YAML string, with resources separated by document
 # markers ("---").
 resources:
-- ../rbac/rbac_role.yaml
-- ../rbac/rbac_role_binding.yaml
-- ../manager/manager.yaml
+- rbac/rbac_role.yaml
+- rbac/rbac_role_binding.yaml
+- manager/manager.yaml
```

Comment:
Can we drop this to minimize changes with the upstream repo?

Reply:
Fixed